content/docs/guides/neon-authorize-google-identity.md

```diff
@@ -0,0 +1,345 @@
+
+## Prerequisites
+
+To follow along with this guide, you will need:
+
+- A Neon account. Sign up at [Neon](https://neon.tech) if you don't have one.
+- To use Google Cloud Platform services, you'll need OAuth credentials. If you haven't set them up, follow the [quickstart](https://developers.google.com/identity/gsi/web/guides/get-google-api-clientid) or these steps.
+  1. Go to [Google Cloud Console](https://console.cloud.google.com/) and create a new project if needed
+  2. Navigate to **APIs & Services > Credentials**
+  <div style={{ display: 'flex', justifyContent: 'center'}}>
+    <img src="/docs/guides/google_oauth_create_credentials.png" alt="Add Authentication Provider" style={{ width: '100%', maxWidth: '750px', height: 'auto' }} />
+  </div>
+  3. Click **Create Credentials** and select **OAuth 2.0 Client ID**
+  4. Set application type to **Web application**
+  5. Add `http://localhost:3000/api/auth/callback/google` to **Authorized redirect URIs**
+     <Admonition type="note" title="Note">
+     Replace the redirect URI with your application's callback URL, depending on your framework and environment.
+     </Admonition>
+  6. Save your `Client ID` and `Client Secret` for later use
+
+## Integrate Google Identity with Neon Authorize
+
+In this first set of steps, we'll integrate Google Identity as an authorization provider in Neon. When these steps are complete, Google Identity will start passing JWTs to your Neon database, which you can then use to create policies.
+
+### 1. Get your Google Identity JWKS URL
+
+When integrating Google Identity with Neon, you'll need to provide the JWKS (JSON Web Key Set) URL. This allows your database to validate the JWT tokens and extract the user_id for use in RLS policies.
+
+The Google Identity JWKS URL is:
+
+```
+https://www.googleapis.com/oauth2/v3/certs
+```
+
+<Admonition type="note" title="Note">
+For the JWT Audience value required by Neon Authorize, you'll use the OAuth client ID which was saved earlier.
+</Admonition>
+
+### 2. Add Google Identity as an authorization provider in the Neon Console
+
+Once you have the JWKS URL, go to the **Neon Console** and add Google Identity as an authentication provider under the **Authorize** page. Paste the JWKS URL and Google Identity will be automatically recognized and selected. Add your Google Cloud project ID as the JWT Audience value and click **Set Up**.
```

Comment:
The admonition above says to use client ID for the audience value, but here, you say to use project ID. Which one is it? (I am guessing the client ID otherwise we probably wouldn't need so much detail in the prereq section, since we assume a user is coming to this with an app that already uses Google Identity for auth)